extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -440,13 +448,17 @@ export function runCandidates(ampdoc, candidates) {
     whenLoaded(candidate).then(() => {
       return measureIntersectionNoRoot(candidate).then(
         ({boundingClientRect}) => {
-          // <amp-img> will change the img's src inline data on unlayout and
-          // remove it from DOM.
-          if (!candidate.signals().get(CommonSignals.LOAD_END)) {
+          if (
+            !candidate.tagName === 'IMG' &&
```

Comment:
Convention from what I've seen:
```suggestion
            !candidate.tagName.toLowerCase() === 'img' &&
```

Side note: I hate this convention/requirement. `toLowerCase` is 11 characters that probably can't even be minified anywhere, and it appears 24 times in v0.js and (if a rough grep is reliable) ~4000 times total in dist. According to MDN, it should always be capitalized for [DOM trees representing HTML documents](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName#:~:text=For%20DOM%20trees%20which%20represent%20HTML%20documents). Maybe we can just do it the way you do above?

Reply:
I'm fine with making this change.